Remove generate installer verbiage and update screenshot (#7111)
fleetdm/fleet: articles/how-to-install-osquery-and-enroll-macos-devices-into-fleet.md

## Installing osquery

> Head over to the Hosts page on Fleet and click on the “Add hosts” button, which will present a pop-up that allows you to choose the type of installer you want to generate. Make sure you are on the “macOS” tab and click on the clipboard icon to copy the installation command for macOS.

Reviewer: ```suggestion
Head over to the Hosts page in Fleet UI and click the “Add hosts” button. From the pop-up, choose the type of installer you want to generate. Make sure you are on the “macOS” tab and click on the clipboard icon to copy the installation command for macOS.
```
